environments/hca-prod/terraform/variables.tf

```diff
@@ -0,0 +1,7 @@
+locals {
```

Comment:
why locals vs var?

Reply:
because I copied and pasted from the dev code and didn't think about it lol